x-pack/plugins/infra/public/components/alerting/logs/log_threshold_alert_type.ts

```diff
@@ -25,5 +25,6 @@ export function getAlertType(): AlertTypeModel {
         defaultMessage: `\\{\\{context.matchingDocuments\\}\\} log entries have matched the following conditions: \\{\\{context.conditions\\}\\}`,
       }
     ),
+    isUiEditEnabled: false,
```

Comment:
For `infra` (both metrics and logs) we’ve refactored the parts of our code that relied on our expression editor rendering inside of our app already. So for us this can be changed to `true` (unless I missed something else 😬). Unless the idea is to blanket disable the Observability plugins alerts until all plugins have made the needed changes for consistency?